pages/quiz.html

```diff
@@ -0,0 +1,333 @@
+                >
+                  play quiz
+                </a>
+                <a
+                  id="modal-btn"
+                  class="btn btn-square-solid btn-xs bg-indigo-500 text-white"
+                >
+                  quiz rules
+                </a>
+
+                <div id="modal" class="modal p-20">
+                  <div class="modal-cont bg-grey-400">
+                    <span
+                      class="modal-close-btn close-btn text-slate-700 text-hover-white fas fa-times"
+                    ></span>
+                    <h2>quiz rules</h2>
+                    <p>
+                      Lorem, ipsum dolor sit amet consectetur adipisicing elit.
+                      Fugiat, veniam amet vel iure nulla expedita quisquam nisi
+                      neque repellendus temporibus consectetur eaque fugit
+                      asperiores aut dicta iusto aspernatur ipsum libero?
+                    </p>
+                  </div>
+                </div>
+              </div>
+            </div>
+          </div>
+        </div>
+      </div>
+    </main>
+    <!-- main section -->
+
+    <div class="spacer-3rem"></div>
+    <div class="spacer-3rem"></div>
+    <div class="spacer-3rem"></div>
+    <div class="spacer-3rem"></div>
+
+    <!-- footer -->
+    <footer
+      class="relative flex flex-col h-44 justify-center align-items-center w-full p-5 z-50 bg-cyan-200"
+      style="bottom: 0"
+    >
+      <div class="text-grey-700">
+        Made with
+        <span class="text-cyan-700 font-bold ml-1 mr-1">&lt;/&gt;</span> by
+
+        <a class="ml-1 text-hover-cyan-600">Ananya</a>
+      </div>
+      <div class="flex mt-3 align-items-center justify-center">
+        <a href="https://github.com/Ananyamadhu08" target="_blank">
+          <i class="fab fa-github mr-2 text-cyan-700"></i>
+        </a>
+        <a href="https://twitter.com/AnanyaMadhu27" target="_blank">
+          <i class="fab fa-twitter mr-2 text-cyan-700"></i>
+        </a>
+        <a
+          href="https://www.linkedin.com/in/ananya-madhu-74479b206/"
+          target="_blank"
+        >
+          <i class="fab fa-linkedin mr-2 text-cyan-700"></i>
+        </a>
+      </div>
+
+      <div class="text-grey-700 mt-5">
+        Designed using
+
+        <a class="text-hover-cyan-600">Fusion CSS</a>
+      </div>
+    </footer>
+    <!-- footer -->
+
+    <script>
+      var modal = document.querySelector("#modal");
+
+      var modalBtn = document.querySelector("#modal-btn");
+
+      var closeBtn = document.querySelector(".modal-close-btn");
+      console.log(closeBtn);
+
+      modalBtn.onclick = function () {
+        modal.style.display = "block";
+      };
+
+      closeBtn.onclick = function () {
+        modal.style.display = "none";
+      };
+
+      window.onclick = function (event) {
+        if (event.target == modal) {
+          modal.style.display = "none";
+        }
+      };
+    </script>
```

Comment:
Question : Why are we writing JS in HTML file . Could we make a separate file for this ?
Also using const or let would be much better. 
Remove commented code